app/src/vanilla/java/cm/aptoide/pt/app/view/EditorialFragment.java

```diff
@@ -316,17 +343,27 @@ public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container,
     }
   }
 
-  @Override public void showDownloadAppModel(DownloadAppViewModel model) {
-    DownloadModel downloadModel = model.getDownloadModel();
+  @Override public void showDownloadModel(DownloadModel downloadModel) {
     this.action = downloadModel.getAction();
+    EditorialItemsViewHolder editorialItemsViewHolder =
```

Comment:
I would recommend a different name for this variable (not for the class). Here, just like everywhere the getViewHolderForAdapterPosition method is called, you could name it something more specific like placeHolderViewHolder for example